src/index.js

```diff
@@ -1,3 +1,46 @@
 import { example } from './example.js';
 
 example();
+
+  
+
+//Comenzar firebase código que te entrega la pág
+var firebaseConfig = {
```

Comment:
var ya no se usa, de preferencia usen const o let
```suggestion
const firebaseConfig = {
```